src/ai/backend/manager/models/kernel.py

```diff
@@ -303,6 +294,14 @@ def default_hostname(context) -> str:
 }
 
 
+SESSION_TYPE_IMAGE_ROLE_MAP: Mapping[SessionTypes, str] = {
+    SessionTypes.BATCH: "COMPUTE",
+    SessionTypes.INTERACTIVE: "COMPUTE",
+    SessionTypes.INFERENCE: "INFERENCE",
+    SessionTypes.SYSTEM: "SYSTEM",
+}
```

Comment:
Why is this needed if we migrate to `enum.StrEnum`?

Reply:
`StrEnum` fields are string values that are also bidirectionally convertible from/to the Enum instances. There is **no need** to have an extraneous mapping from StrEnum instances to str values **at all.**

Reply:
Read the manual: https://docs.python.org/3/library/enum.html#enum.StrEnum

Reply:
Ah, this mapping is for:
<img width="587" alt="image" src="https://github.com/user-attachments/assets/92beedd3-dab2-46f0-b5ea-3d49fb97eb20">

It seems to be an equivalent but unnecessarily reversed logical expression.
It should be a mapping from the label value to a `SessionTypes` value and compared with `session_type`.